docs/docs-source/docs/modules/run-cloudflow-gcp-marketplace/pages/deploy-cloudflow.adoc

```diff
@@ -0,0 +1,45 @@
+= Deploy Cloudflow on GCP Marketplace
+:toc:
+:toc-title: ON THIS PAGE
+:toclevels: 2
+
+== Deploying Cloudflow
+
+You have followed the xref:find-cloudflow-on-gcp-marketplace.adoc[previous step] of locating Cloudflow in the Marketplace catalog and now see the following screen:
+
+.The Cloudflow page on Marketplace
+image::cloudflow-main-gcp.png[Cloudflow page on Marketplace]
+
+Spend some time reading through the information on the page. When you are ready, click "Configure" to proceed to the deployment screen:
+
+.The Cloudflow Configuration and Deployment Page
+image::deploy.png[Cloudflow deployment page]
+
+On this page, you have the ability to configure a few parameters before deploying Cloudflow. First, you'll need a Google Kubernetes Engine (GKE) cluster.
+If you have an existing cluster, you may select it from the dropdown menu.
+Or you can quickly spin up a new cluster in your preferred cluster zone by clicking the "Create cluster" button.
+Note that your existing cluster may not be available for selection if it does not meet the resource requirements, in which case you'll also need to create a new cluster.
+
+Once your cluster is up and running, go through the other configuration options.
+Some of them have a tooltip next to them that explains the purpose of that configuration option.
+
+* Namespace:
+The namespace where Cloudflow will be installed
+* App instance name: The name of the Cloudflow installation.
+The Cloudflow installer will create a custom resource of this name that is of kind `cloudflows.cloudflow-installer.lightbend.com` (or `cloudflow` for short), a CustomResourceDefinition defined by the installer.
+The instantiation of this custom resource will then trigger the installation of other Cloudflow components.
+* `initImage` and `installerImage`
+These are images needed to initialize and launch the installer. Make sure their tags match your desired Cloudflow version.
```

Comment:
Is this set correctly by default? The "Make sure their tags match..." implies they have to take some action.

Reply:
What I noticed is that there could be mismatch as shown in the default settings:

<img width="406" alt="Screen Shot 2020-06-24 at 09 57 48" src="https://user-images.githubusercontent.com/1168769/85570427-55dd2400-b601-11ea-9b99-68bdfb07641b.png">

where the `gcr.io/lightbend-public/cloudflow` is the same as `lightbend/cloudflow-installer` image. It's not a problem in this specific case because `gcr.io/lightbend-public/cloudflow:2.0.0` and `gcr.io/lightbend-public/cloudflow:2.0.1` are identical.

Reply:
Good point!